geckomat/utilities/FVA/comparativeFVA.m

```diff
@@ -0,0 +1,138 @@
+function [FVA_Dists,indexes,stats] = comparativeFVA(model,ecModel,c_source,chemostat,tol,blockedMets)
+% comparativeFVA
+%  
+% This function goes through each of the rxns in a metabolic model and
+% gets its flux variability range, then the rxn is mapped into an EC
+% version of it to perform the correspondent variability analysis and
+% finally compares and plots the cumulative flux variability distributions. 
+%
+%   model       MATLAB GEM structure, constrained with the desired culture
+%               medium
+%   ecModel     MATLAB ecGEM structure, constrained with the desired culture
+%               medium
+%   c_source    rxnName for the main carbon source uptake reaction 
+%   chemostat   TRUE if chemostat conditions are desired
+%   blockedMets metNames of metabolites which secretion should be blocked
```

Comment:
any example of said metabolites? Why have them as an extra input and not instead tell the user to change them beforehand in the model? I thought you already specified that in [`5277282`](https://github.com/SysBioChalmers/GECKO/pull/57/commits/5277282a7a8931b4fba96abcc9ac6d3f9446ef44)